File: Basic/code/problem_10.c
/*
Write a program to Input 5 digit numbers and calculate sum of its digits.
*/

// Solved by napalion

#include<stdio.h>
int main(){
    int n,sum,copy;
    sum = 0;
    printf("Enter a 5 number digits: \n");
    scanf("%d",&n);
    copy = n; //copy the number to make sure we don't lose it.
    for(int i=0;i<5;i++){
        int y = copy%10; //digit
        copy = copy/10; //number without digit y
        sum = sum + y; //add digit to sum
    }
    printf("The sum of digits of the number %d is = %d\n",n,sum);
    return 0;
}

File: Basic/code/problem_10.py
# 10. Input 5 digit numbers and calculate sum of its digits.
# by Nitin Kumar

sum=0
n1 = int(input('Enter a 5-digit number:'))
while n1:
    i = n1%10
    n1=int(n1/10)
    sum+=i
print(sum)


//By Parth Thaker


def getSum(n):
     
    strr = str(n)
    list_of_number = list(map(int, strr.strip()))
    return sum(list_of_number)
   
n = 12345
print(getSum(n))
